pulsebot/bugzilla.py

```diff
@@ -21,31 +25,45 @@ def get_fields(self, bug, fields):
         bug_url = "%s/rest/bug/%d?include_fields=%s" % (
             self._server,
             bug,
-            "+".join(fields),
+            ",".join(fields),
         )
+        logger.info(f"get_fields: {bug_url}")
         try:
             r = self._session.get(bug_url)
             r.raise_for_status()
             bug_data = r.json()
         except Exception:
+            logger.exception(
+                f"Error occurred retrieving bug fields {bug_url} {r.status_code}"
+            )
             raise BugzillaError()
 
         if "error" in bug_data:
+            logger.exception(
```

Comment:
Technically there is no exception prior to this log entry here, so this should be `logger.error`. 
```suggestion
            logger.error(
```